internal/services/housekeeper/postgresPartitionCleaner.go

```diff
@@ -0,0 +1,32 @@
+package housekeeper
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/redhatinsights/ros-ocp-backend/internal/config"
+	database "github.com/redhatinsights/ros-ocp-backend/internal/db"
+)
+
+func DeletePartitions() {
+	// log := logging.GetLogger()
+	cfg := config.GetConfig()
+	db := database.GetDB()
+	currentTime := time.Now()
+
+	beforeDate := currentTime.AddDate(0, 0, -cfg.DataRetentionPeriod)
```

Comment:
Can we name these variables as more descriptive?  Renaming `beforeDate` to `retentionThresholdDate` and `table_date` as `partitionDropDate` or `partitionTableDate` or any other suggestions?